target/linux/ath79/dts/ar9330_openmesh_om2p-v2.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9330_openmesh_om2p-lc.dtsi"
```

Comment:
With om2p-lc and om2p-v2 sharing this as the only devices on that soc, I'd call the DTSI ..._om2p.dtsi

Reply:
Not sure about the name. But I also was not sure about the old name - so consider it done